public/static/dist/css/styles.css

```diff
@@ -544,11 +544,11 @@ md-select:not([disabled]):focus .md-select-value {
 md-input-container label {
   font-size: medium;
 }
-md-radio-button .md-on,
+md-radio-button:not([disabled]) .md-on,
```

Comment:
Can `:not([disabled])` be rewritten as `:enabled`?

Reply:
I wasn't able to get it working without the double negative :/